test/screenshot/text-field/TextFieldPermutations.tsx

```diff
@@ -25,37 +26,41 @@ const textFields = (variantProps: {variant?: string}) => {
           return requiredMap.map((isRequired: {required?: boolean}) => {
             return disabledMap.map((disabled: {disabled?: boolean}) => {
               return helperTextMap.map((helperText: HelperTextMapType | {}) => {
-                const isValidationTextField = isValidationMsg(helperText);
-                const value = !isValidationTextField
-                  ? {value: null}
-                  : {value: ''};
-                const iconKey = Object.keys(icon)[0] || '';
-                const denseKey = Object.keys(dense)[0] || '';
-                const rtlKey = Object.keys(isRtl)[0] || '';
-                const disabledKey = Object.keys(disabled)[0] || '';
-                const isRequiredKey = Object.keys(isRequired)[0] || '';
-                const helperTextKey = getHelperKeyText(
-                  helperText,
-                  isValidationTextField
-                );
-                const props = Object.assign(
-                  {},
-                  variantProps,
-                  icon,
-                  dense,
-                  disabled,
-                  helperText,
-                  isRequired,
-                  isRtl,
-                  value
-                );
-                const key = `${iconKey}-${denseKey}-${disabledKey}-${helperTextKey}-${isRequiredKey}--${rtlKey}`;
-                const hasIcon =
-                  iconKey === 'leadingIcon' || iconKey === 'trailingIcon';
-                if (hasIcon && variantProps.variant === 'fullWidth') {
-                  return;
-                }
-                return <TestField {...props} key={key} id={key} />;
+                return noLabelMap.map((noLabel: {noLabel?: boolean}) => {
```

Comment:
Can you move this to the no label to its own screenshot test page? This screenshot image is getting longer/larger, and want to avoid this slowing down the screenshot tests.

Sorry for the inconvenience